src/commands/publish/upload_form/mod.rs

```diff
@@ -57,7 +57,7 @@ pub fn build(
         }
         TargetType::Webpack => {
             log::info!("Webpack project detected. Publishing...");
-            // FIXME(sven): shouldn't new
+            // TODO: (sven) shouldn't new
```

Comment:
@xtuc could you explain this comment?

Reply:
i think this should actually be above ln 62, where we call `wranglerjs::Bundle::new` but i'm not certain precisely why.

Reply:
My idea was that each backends (rust, webpack, js) returns a Bundle/WorkerBundle/Worker that contains all the bindinds. The creation of the form (code here), would consume it and build a form regardless of the backend used.

If that was implemented building a new bundle wouln't make sense. However, wranglerjs's bundle doesn't contains any state (only constants) so it's fine.

Feel free to remove the comment tho. We could also inline the constants for now.